### PR TITLE
402 web global error handling

### PR DESCRIPTION
**Pull request by CedricCORGE**
- Add custom messages for error in back
- Use theses messages in the web app

### CODE REVIEW

File: Server/controllers/Common/container.js [hunk 1]
```diff
@@ -6,54 +6,69 @@ const { db } = require("../../middleware/database");
...
-exports.getContainerById = (id) => {
-  let idtest = parseInt(id);
-  return db.Containers.findUnique({
-    where: { id: idtest },
-    select: {
-      city: true,
-      address: true,
-      items: {
-        where: { available: true },
+exports.getContainerById = async (id) => {
```
Comment (31Nathan) [inline Server/controllers/Common/container.js:9]: Missing throw on doc, check the files for similarities

File: Server/controllers/Common/itemCategory.js [hunk 2]
```diff
@@ -1,67 +1,87 @@
-const { db } = require('../../middleware/database')
+const { db } = require("../../middleware/database");
```
Comment (31Nathan) [inline Server/controllers/Common/itemCategory.js:2]: Same here

File: Server/controllers/Common/tickets.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const { db } = require('../../middleware/database')
+const { db } = require("../../middleware/database");
 
 /**
  * Create a new ticket in the database
```
Comment (31Nathan) [inline Server/controllers/Common/tickets.js:4]: Again

File: Server/controllers/Web/feedbacks.js [hunk 4]
```diff
@@ -6,10 +6,16 @@ const { db } = require('../../middleware/database')
  * @param {*} data of the feedback to be created
  * @returns the newly created feedback
  */
```
Comment (31Nathan) [inline Server/controllers/Web/feedbacks.js:8]: Again

File: Server/controllers/Web/organization.js [hunk 5]
```diff
@@ -6,10 +6,14 @@ const { db } = require("../../middleware/database");
  * @param {*} organization
```
Comment (31Nathan) [inline Server/controllers/Web/organization.js:6]: Check every file

**Review by hugolopespinto — APPROVED** [review]

**Review by 31Nathan — APPROVED** [review]

**Pull request merged**